docker-compose.yml

```diff
@@ -17,20 +17,28 @@ services:
       MYSQL_DATABASE: prestashop
     restart: always
   prestashop-git:
-    build: .docker
+    build:
+      dockerfile: .docker/Dockerfile
+      context: .
+      args:
+        - USER_ID=${USER_ID:-1000}
+        - GROUP_ID=${GROUP_ID:-1000}
     environment:
       DISABLE_MAKE: ${DISABLE_MAKE:-0}
       PS_INSTALL_AUTO: ${PS_INSTALL_AUTO:-1}
       DB_PASSWD: ${DB_PASSWD:-prestashop}
       DB_NAME: ${DB_NAME:-prestashop}
       DB_SERVER: ${DB_SERVER:-mysql}
+      DB_PREFIX: ${DB_PREFIX:-ps_}
       PS_DOMAIN: ${PS_DOMAIN:-localhost:8001}
       PS_FOLDER_INSTALL: ${PS_FOLDER_INSTALL:-install-dev}
       PS_FOLDER_ADMIN: ${PS_FOLDER_ADMIN:-admin-dev}
+      PS_COUNTRY: ${PS_COUNTRY:-fr}
+      PS_LANGUAGE: ${PS_LANGUAGE:-fr}
+      PS_DEV_MODE: ${PS_DEV_MODE:-0}
+      ADMIN_PASSWD: ${ADMIN_PASSWD:-prestashop_demo}
     command: ["/tmp/wait-for-it.sh", "--timeout=60", "--strict", "mysql:3306", "--", "/tmp/docker_run_git.sh"]
     ports:
       - "8001:80"
     volumes:
-      - ./:/var/www/html/:delegated
-      - vendor:/var/www/html/vendor
```

Comment:
Why do you remove `var` and `vendor` volumes ? I think it would be really heavy for dev purpose. (I mean re-running composer install for each container restart doesn't look really efficient ^^')

Reply:
For me it's already included on `./:/var/www/html/:delegated` and by passing `DISABLE_MAKE=1` you can disable `composer install` and `build assets` scripts. but I'm maybe wrong

Reply:
From this article, `delegated` and `cached` are not good for prestashop dev purpose cause most of source files have to be edited from host and container create / edit lot of files as well.

https://tkacz.pro/docker-volumes-cached-vs-delegated/

I recommend more tests but I use my own docker-compose so if is QA only, I let you the choice :thinking: :yum:

Reply:
The docker-compose is not for QA only, it could be used by everyone. If you want, we will wait for other reviewers opinion and delete this update if necessary.